include/open62541/plugin/eventloop.h

```diff
@@ -555,6 +555,25 @@ UA_ConnectionManager_new_POSIX_UDP(const UA_String eventSourceName);
 UA_EXPORT UA_ConnectionManager *
 UA_ConnectionManager_new_POSIX_Ethernet(const UA_String eventSourceName);
 
+
+/**
+ * HTTP Connection Manager
+ * ~~~~~~~~~~~~~~~~~~~~~~~
+ *
+ * The HTTP ConnectionManager uses the libwebsockets library to send http requests.
+ *
+ * Open Connection Parameters:
+ * - 0:hostname [string]: Hostname (or IPv4/v6 address) of the connection (required).
+ * - 0:port [uint16]: Port of the target host (default: 443).
+ * - 0:path [string]: Path is the string of information that comes after the top level domain name (default: /).
```

Comment:
The way I understand it, the connection will stay open for multiple requests.
The path, method and data should be parameters for `sendWithConnection`, not here.